test/spmd/tensor/test_tp_sharding_ops.py

```diff
@@ -41,7 +41,8 @@ def test_sharded_transpose(self):
     # TODO: Need to investigate why test failed in CPU for baddbmm.
     @with_comms
     @skip_if_lt_x_gpu(TEST_GPU_NUM)
-    def test_sharded_baddbmm(self):
+    def test_sharded_baddbmm_beta_zero(self):
```

Comment:
Can we merge the test of these two? Thanks!

Reply:
Will it be good to move the tests into `test_matrix_ops.py` since these ops are now in `matrix_ops`?

Reply:
Sounds good to me.